### ISSUE TITLE
Ai Bodyguard Unit stance.

### ISSUE THREAD

**Issue by ilosevsrat** [Sh0wdown/UnofficialCrusaderPatch#776]
Hi All!

we all know that Ai"s starting melee troops units (bodyguards) are usually have idle stance and not reacting to anything, i think changing their stance to Aggressive or Defensive stance would be appropriate since their main job is to defend Lord and not stand idle when enemy swarm comes to kill HIM =D.

its just immersion improving stuff that i think needs to be adressed.

**Comment by J-T-de** [comment]
Iirc, the starting units count to the defensive unit groups. If there are spots for the corresponding units in the AIV, the AI will send units to these spots (and they will do their intended thing). Starting units, which have no spot in the AIV will stay and chill on the keep. I think @Krarilotus once madet his observation, can you confirm?

So I see two possible ways to fix this:
- fix the aiv? (one could fix this problem by just set a unit spot to the campfire)
- fix this ingame: each (melee-) starting unit without a aiv spot needs to be set to aggressive. The aiv spots are actually known, but setting the unit stance I have no idea...

**Comment by ilosevsrat** [comment]
yes, starting troops are acting according to set-up in AIV, (except  Swordsman, Pikeman and few other troops that will not go to pointed defensive positions that is set in AIV, AFAIR) its bugged i believe. 

possible solution is to set starting troops to defensive or aggressive stance trough scripting (aggressive is preferable, since they count and definitely look like  lords personal bodyguards., they should React more aggressively to any potential threat that comes near lord. :)

**Comment by J-T-de** [comment]
So I guess fixing https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/318 might also fix this issue. Just as a cross reference for the future...

**Comment by GRhin** [comment]
I think defensive stance would be better than aggrressive, so they cant be drawn away from the lord easier. Couple this with #642 and i think its much more effective
